packages/@vuepress/core/lib/node/ClientComputedMixin.js

```diff
@@ -87,11 +87,13 @@ module.exports = siteData => {
         page.frontmatter.title // explicit title
         || page.title // inferred title
       )
+      const description = this.$description.trim()
+      const titleDescription = description.length > 0 ? siteTitle + ' | ' + description : siteTitle
```

Comment:
What the point of using `siteTitle`? It will always be null here.

Reply:
Thanks for pointing out. Fixed.